helper_dockerfiles/Dockerfile-grype

```diff
@@ -1,11 +1,24 @@
 # Get Python Image
 FROM --platform=linux/amd64 public.ecr.aws/bitnami/python:3.11
+ARG OFFLINE="NO"
+ARG OFFLINE_SEMGREP_RULESETS=""
+
+ENV OFFLINE="${OFFLINE}"
+ENV OFFLINE_SEMGREP_RULESETS="${OFFLINE_SEMGREP_RULESETS}"
+ENV GRYPE_DB_CACHE_DIR="${HOME}/.grype"
 
 # Instal prerequisites
 RUN curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -s -- -b /usr/local/bin && \
     curl -sSfL https://raw.githubusercontent.com/anchore/grype/main/install.sh | sh -s -- -b /usr/local/bin && \
     python3 -m pip install semgrep
 
+RUN RUN if [[ "$OFFLINE" == "YES" ]]; then \
```

Comment:
Double `RUN` call here

Reply:
Also worth considering that this Dockerfile is going away soon; I think just correcting this one bit should be fine so it's syntactically correct, but I wouldn't spend too much more time on this one :-)